velox/row/tests/UnsafeRowFuzzTest.cpp

```diff
@@ -118,6 +120,8 @@ TEST_F(UnsafeRowFuzzTests, fast) {
       ARRAY(VARCHAR()),
       ARRAY(VARBINARY()),
       ARRAY(UNKNOWN()),
+      ARRAY(DECIMAL(20, 2)),
```

Comment:
FYI, Fuzzer doesn't generate complex types with nulls: https://github.com/facebookincubator/velox/issues/6195

You may want to apply the fix suggested in that issue and re-run the test to see if anything fails.